lib/wovnrb/html_replacers/link_replacer.rb

```diff
@@ -18,7 +18,7 @@ def replace(dom, lang)
 
         href = node.get_attribute('href')
         next if href =~ /^\s*\{\{.+\}\}\s*$/
-        next if href =~ /javascript:void\(/
+        next if href =~ /^\s*javascript:/
```

Comment:
Could you add case insensitive to regular expression?
As `"JAVASCRIPT:alert(1)" =~ /^\s*javascript:/i`.

Reply:
I will, thanks

Reply:
done, thanks